mapswipe-dashboard/app/components/ItemSelectInput/index.tsx

```diff
@@ -86,12 +86,13 @@ function ItemSelectInput<Name extends string>(props: ItemSelectInputProps<Name>)
     }), [debouncedSearchText]);
 
     const location = useLocation();
+    const pathName = location.pathname.split('/');
 
     useEffect(() => {
-        if (location.pathname === '/' && isDefined(selectedItem)) {
+        if ((location.pathname === '/' && isDefined(selectedItem)) || (selectedItem && !pathName.includes(selectedItem))) {
```

Comment:
Need to discuss more on this

Reply:
this is a hack way to clear selected item when navigation changes by clicking on links